include/crpropa/module/Observer.h

```diff
@@ -241,12 +241,21 @@ class ObserverParticleIdVeto: public ObserverFeature {
  */
 class ObserverTimeEvolution: public ObserverFeature {
 private:
-	std::vector<double> detList;
+	int numb;
+	bool islog = false, useCustomGetTime = false;
+	double min, max;
 public:
+	// List containing all used times, is constructed in constructor or by user manually
+	// (leave empty if you want to rather use functions)
+	std::vector<double> detList;
```

Comment:
detList is here defined public since it is necessary that the user has full control over it